deb/changes.go

```diff
@@ -189,6 +189,9 @@ func (c *Changes) PackageQuery() (PackageQuery, error) {
 		binaryQuery = &AndQuery{
 			L: &NotQuery{Q: &FieldQuery{Field: "$PackageType", Relation: VersionEqual, Value: "source"}},
 			R: binaryQuery}
+		binaryQuery = &OrQuery{
+			L: &FieldQuery{Field: "Name", Relation: VersionPatternMatch, Value: "*-dbgsym"},
```

Comment:
I think this breaks `.changes` consistency: aptly verifies that `.changes` is valid and signed by known key, so it tried to avoid pulling packages which are not referenced in `.changes`. This query matches any `*-dbgsym` package.